internal/ui/json/backup.go

```diff
@@ -107,6 +108,9 @@ func (b *Backup) Run(ctx context.Context) error {
 	for {
 		select {
 		case <-ctx.Done():
+			if errorCount > 0 {
+				return errors.Errorf("Errors occured during backup.")
```

Comment:
What about "Failed to read all source data during backup"? It is more descriptive.